actors/evm/src/interpreter/system.rs

```diff
@@ -122,6 +122,24 @@ impl<'r, RT: Runtime> System<'r, RT> {
         self.rt.send_read_only(to, method, params)
     }
 
+    /// Generalized send
+    pub fn send_with_gas(
+        &mut self,
+        to: &Address,
+        method: MethodNum,
+        params: RawBytes,
+        value: TokenAmount,
+        gas_limit: Option<u64>,
```

Comment:
Maybe make this take a `U256` so we can handle "ethereum gas logic" everywhere?

Reply:
I don't think we should make U256 show up with all its ugliness here, better to add a truncating from method.

Reply:
Added `to_u64_saturating`